src/web/components/Card/components/CardListItem.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { css, cx } from 'emotion';
+
+import { palette } from '@guardian/src-foundations';
+
+const listStyles = (percentage?: CardPercentageType) => css`
+    display: flex;
+    flex-grow: 1;
+
+    /* Here we ensure the card stretches to the given percentage */
+    flex-basis: ${percentage && percentage};
+
+    /* We absolutely position the vertical divider and this contains that */
+    position: relative;
+
+    /* Set spacing on the li element */
+    padding-left: 10px;
```

Comment:
I forgot what the conversations were about using the spacing tokens from the design system?

Reply:
That they were good and we should use them but at the time they weren't available. I think they recently landed though, I saw an email from @SiAdcock . I'm expecting to have a big spacing PR where we transfer the world and enforce going forward.

Reply:
Sounds good.